src/com/yetanalytics/lrs_admin_ui/views/browser.cljs

```diff
@@ -20,7 +21,11 @@
 
 (defn browser []
   (let [content @(subscribe [:browser/get-content])
-        credentials @(subscribe [:db/get-credentials])]
+        credentials @(subscribe [:db/get-credentials])
+        ;;filter out credentials that can't read the LRS
+        read-credentials (filter #(some #{"all" "all/read" "statements/read"}
+                                        (:scopes %))
```

Comment:
This looks like a bug waiting to happen in case we ever change the allowed scopes downstream. You should at least separate out the scope set as its own `def` so it's less buried in the middle of the code.